Tasks/ServiceFabricUpdateAppVersions/Update-PackageVersion.psm1

```diff
@@ -77,7 +77,7 @@
                     # Search the package files for differences (Unless the user doesn't want to log all changes and we already know the package xml changed)
                     $newPackagePath = Join-Path $NewPackageRoot $packageName
                     $oldPackagePath = Join-Path $OldPackageRoot $packageName
-                    if (Find-FileChanges $newPackagePath $oldPackagePath $LogIndent -LogAllChanges:$LogAllChanges)
+                    if ((Test-Path $newPackagePath) -and (Find-FileChanges $newPackagePath $oldPackagePath $LogIndent -LogAllChanges:$LogAllChanges))
```

Comment:
That's true.  If a new package is being added where there was no previous version of that package, then it will call Find-FileChanges and fail validation because the old package path doesn't exist.  And in that case, we actually don't need to update the package version; there's no need since there's no old package that exists that it needs to differ from in order to upgrade.  So here's the decision matrix:
Old Package Exists: False
New Package Exists: False
Update Package Version: False

Old Package Exists: True
New Package Exists: False
Update Package Version: False

Old Package Exists: False
New Package Exists: True
Update Package Version: False

Old Package Exists: True
New Package Exists: True
Update Package Version: True if there are file changes between packages